packages/salesforcedx-utils-vscode/src/services/sourceTrackingService.ts

```diff
@@ -33,7 +33,7 @@ export class SourceTrackingService {
       project,
       ignoreLocalCache: true,
       subscribeSDREvents: true,
-      ignoreConflicts: false
+      ignoreConflicts: true
```

Comment:
VSCE will not use STL for conflict detection.  Instead it will continue to use the logic behind the "Detect Conflicts at Sync" setting with uses the TimestampConflictDetector.

Until there is a way to deploy and override any conflicts in STL (when STL was already instantiated in the running process with `ignoreConflicts` set to `true`, we will need to continue to check for conflicts _before_ running the deploy operation (as opposed to catching the STL exception thrown mid-deploy operation)